haas/api.py

```diff
@@ -422,10 +441,15 @@ def headnode_create_hnic(headnode, hnic):
     _assert_absent_n(db, headnode, model.Hnic, hnic)
 
     if not headnode.dirty:
-        raise IllegalStateError
+        raise IllegalStateEr
```

Comment:
Yep, that is an unintentional typo. Do I have to do anything special to fix it while the code has a pull request, or will it work if I just push the change to the same branch?